users/models.py

```diff
@@ -20,6 +20,11 @@ class User(AbstractUser):
         verbose_name='User\'s photo',
         blank=True)
     zip_code = models.CharField(max_length=12, blank=True)
+    is_active = models.BooleanField(
```

Comment:
It's better to add a separate field named "email_confirmed" or smth like that. "is_active" can be used to delete users